builtins/davix/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+include(ExternalProject)
+
+set(DAVIX_VERSION "0.6.4")
+set(DAVIX_URL "http://grid-deployment.web.cern.ch/grid-deployment/dms/lcgutil/tar/davix")
+set(DAVIX_URLHASH "SHA256=4db74681ab83307c5477d29f0680953f1e6359efed001d52a6e8cff47291165b")
+set(DAVIX_PREFIX ${CMAKE_CURRENT_BINARY_DIR}/DAVIX-prefix)
+set(DAVIX_LIBNAME ${CMAKE_STATIC_LIBRARY_PREFIX}davix${CMAKE_STATIC_LIBRARY_SUFFIX})
+
+foreach(lib davix neon boost_static_internal)
+  set(libname ${CMAKE_STATIC_LIBRARY_PREFIX}${lib}${CMAKE_STATIC_LIBRARY_SUFFIX})
+  list(APPEND DAVIX_LIBRARIES ${DAVIX_PREFIX}/lib/${libname})
+endforeach()
+
+ExternalProject_Add(DAVIX
+  URL ${DAVIX_URL}/${DAVIX_VERSION}/davix-embedded-${DAVIX_VERSION}.tar.gz
+  URL_HASH ${DAVIX_URLHASH}
+  PATCH_COMMAND patch -p1 -i ${CMAKE_CURRENT_SOURCE_DIR}/davix-0.6.4.patch
+  CMAKE_CACHE_ARGS -DCMAKE_PREFIX_PATH:STRING=${OPENSSL_PREFIX}
+  CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
```

Comment:
Is `<INSTALL_DIR>` in `-DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>` a thing?

Reply:
Yes, it's the temporary installation directory where CMake's `ExternalProject_Add()` puts it.
From https://cmake.org/cmake/help/latest/module/ExternalProject.html:

> The command line, comment, working directory and byproducts of every standard and custom step are processed to replace the tokens <SOURCE_DIR>, <SOURCE_SUBDIR>, <BINARY_DIR>, <INSTALL_DIR> <TMP_DIR>, <DOWNLOAD_DIR> and <DOWNLOADED_FILE> with their corresponding property values defined in the original call to ExternalProject_Add().